integration/c/CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 project(c_otterbrix C CXX)
 
 
+set( ${PROJECT_NAME}_headers
```

Comment:
Why you need those changes? Why you decide to change something in C integration?

Reply:
C# integration uses C files